### PR TITLE
[Unitary] Create overview.md

### PR DESCRIPTION
**Pull request by dstrain115**
Create the overview of unitary from the document created by @cognigami

### CODE REVIEW

File: docs/unitary/overview.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Again, each `QuantumObject` only has a definite, classical state after you retrieve it from the `pop` method. 
+
+At any time before you `pop` the classical state from a `QuantumWorld` object, you can retrieve a sample classical state by using the `peek` method. The `QuantumWorld` can return as many samples from your quantum state as you find useful. You can also retrieve a summary of the probability distribution of the state of your Quantum Object by calling the `get_histogram` method. You should use the `pop` method to score your game state, and you can use the `peek` method to provide information to your players.
+
+In the real world, if you observe a single quantum state configuration twice, you might see two different classical states. To support consistent evolution of game play, the `QuantumWorld` snapshots the classical state after you call `pop`. You can continue to add quantum effects to your `QuantumWorld`, but your new quantum effects will use the classical state snapshot as the starting state for all of your quantum objects. 
+
+## Sampler
```
Comment (cognigami) [inline docs/unitary/overview.md:65]: Remove space.

**Review by cognigami — CHANGES_REQUESTED** [review]

File: docs/unitary/overview.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+A **Quantum World** defines the scope for your quantum effects. In a classical game, the onus is on you to write code to define how game objects relate to each other, according to the model of your game world. In a quantum game, any set of quantum objects could become entangled with each other, depending on the quantum effects that you applied to your quantum objects. When calculating the outcome of quantum effects, the quantum engine needs access to the complete set of quantum objects, in principle. The `QuantumWorld` object acts as a container for the collection of all quantum objects and quantum effects on behalf of the quantum engine.
+
+A single quantum world represents a multiverse of possibilities; we call that multiverse the quantum state. Regular game play should add quantum effects to the quantum world, where the effects apply to one or more quantum objects in the world. As game play progresses, you should build up the quantum state: quantum effects can split and entangle your quantum objects.
+
+## Measurement
```
[the remark this answers is not in the document]
Comment (dstrain115) [reply docs/unitary/overview.md:47]: I actually did the opposite and added spaces where they wer missing above.  The extra space is needed for markdown to denote a title style.

File: docs/unitary/overview.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+One of your tasks as the game designer is to decide when you need to “score” the game. A scoring event could be based on any intermediate classical state that makes sense for your game mechanic. For example, a game might generate a score at the end of a battle, the end of a round, or the end of the game.
+
+When you need to score the game, you should use the `pop` method to retrieve the classical state from your `QuantumWorld` object. You can use the classical state to calculate the appropriate score for your game. 
+
+## Quantum state and classical state
```
[the remark this answers is not in the document]
Comment (dstrain115) [reply docs/unitary/overview.md:53]: Same

File: docs/unitary/overview.md (hunk 1 above)
Comment (dstrain115) [reply docs/unitary/overview.md:65]: Same.

**Review by cognigami — APPROVED** [review]

**Pull request merged**